### PR TITLE
Add some PGPOOL_* directives with default values

### PR DESCRIPTION
**Pull request by editorfatih**
**Description of the change**

The following environment directives have been added with default values;

PGPOOL_ENABLE_POOL_HBA=yes
PGPOOL_ENABLE_POOL_PASSWD=yes
PGPOOL_PASSWD_FILE=pool_passwd
PGPOOL_MAX_POOL=15

**Benefits**

disabling to use of pool_passwd file 
increase/decrease max_pool settings via environment variable

**Possible drawbacks**

---

Applicable issues

---

Additional information

----

### CODE REVIEW

**Comment by andresbono** [comment]
Hi @editorfatih, we will review your PR shortly. Thanks for your contribution!!

File: 4/debian-10/docker-compose.yml [hunk 1]
```diff
@@ -46,6 +46,9 @@ services:
       - PGPOOL_ADMIN_USERNAME=admin
       - PGPOOL_ADMIN_PASSWORD=adminpassword
       - PGPOOL_ENABLE_LOAD_BALANCING=yes
+      - PGPOOL_ENABLE_POOL_HBA=on
```
Comment (andresbono) [inline 4/debian-10/docker-compose.yml]: If these are the default values, we prefer skipping these variables from the `docker-compose.yml`.

File: 4/ol-7/docker-compose.yml [hunk 2]
```diff
@@ -46,6 +46,9 @@ services:
       - PGPOOL_ADMIN_USERNAME=admin
       - PGPOOL_ADMIN_PASSWORD=adminpassword
       - PGPOOL_ENABLE_LOAD_BALANCING=yes
+      - PGPOOL_ENABLE_POOL_HBA=on
```
Comment (andresbono) [inline 4/ol-7/docker-compose.yml]: Same comment than before, could you remove the environment variables?

File: 4/debian-10/docker-compose.yml (hunk 1 above)
Comment (editorfatih) [reply 4/debian-10/docker-compose.yml]: Okay, i did removed these variables from compose file

File: 4/ol-7/docker-compose.yml (hunk 2 above)
Comment (editorfatih) [reply 4/ol-7/docker-compose.yml]: Okay, removed

**Comment by editorfatih** [comment]
Hi @andresbono, i did removed these variables from compose files

File: 4/ol-7/rootfs/libpgpool.sh [hunk 3]
```diff
@@ -283,13 +301,34 @@ EOF
 pgpool_create_config() {
     local -i node_counter=0
     local load_balance_mode=""
+    local allow_clear_text_frontend_auth="off"
```
Comment (andresbono) [inline 4/ol-7/rootfs/libpgpool.sh]: Same comment than above. Can you also declare the variables `pool_hba` and `pool_passwd`?

File: 4/debian-10/rootfs/libpgpool.sh [hunk 4]
```diff
@@ -283,13 +301,35 @@ EOF
 pgpool_create_config() {
     local -i node_counter=0
     local load_balance_mode=""
+    local allow_clear_text_frontend_auth="off"
```
Comment (andresbono) [inline 4/debian-10/rootfs/libpgpool.sh]: Can you also declare the variables `pool_hba` and `pool_passwd`?

**Comment by andresbono** [comment]
@editorfatih, we are going to do some testing of your changes and we will get back to you. At first sight, the changes looks good!

File: 4/ol-7/rootfs/libpgpool.sh (hunk 3 above)
Comment (editorfatih) [reply 4/ol-7/rootfs/libpgpool.sh]: I think that is not neccessary but i did add.

File: 4/debian-10/rootfs/libpgpool.sh (hunk 4 above)
Comment (editorfatih) [reply 4/debian-10/rootfs/libpgpool.sh]: declared

**Comment by editorfatih** [comment]
@andresbono Did you complete your tests, can i create new PR for another changes ?

**Comment by andresbono** [comment]
> @andresbono Did you complete your tests, can i create new PR for another changes ?

Hi @editorfatih, we needed to apply some minor fixes to your configuration changes and we are still testing them.

Is it ok if I push those changes to `editorfatih/bitnami-docker-pgpool` once they are ready? That way you can review them too and we will be able to merge it.

After that, you can open your new PR.

Does it sound good to you?

**Comment by editorfatih** [comment]
@andresbono I'm okay, you push changes and i will review as soon as posibble

**Comment by andresbono** [comment]
@editorfatih, @rafariossaa, will you take a look at the latest changes? Thanks!

**Comment by editorfatih** [comment]
@andresbono i did update your changes, prevent postgres administrator credentials generation if $PGPOOL_ENABLE_POOL_PASSWD is yes, can you verify my changes again ?

**Comment by andresbono** [comment]
Thanks for your contribution, @editorfatih!

**Pull request merged**